inference-engine/tools/cross_check_tool/cross_check_tool.py

```diff
@@ -100,17 +100,17 @@ def get_model_info(net: IENetwork):
 ###
 
 
-@error_handling('processing inference on \'{device}\' device')
+@error_handling('processing inference executable_network')
```

Comment:
The phrase sounds a little bit awkward for me. I think, it'd be better to remove ```executable_network```